docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/wysiwyg/common/cgdp-embed-pullquote/index.scss

```diff
@@ -0,0 +1,63 @@
+@use 'uswds-core' as *;
+@use 'sass:meta' as meta;
+
+.cgdp-embed-pullquote {
+	@include u-measure(6);
+	@include u-padding-y(1.5);
+	@include u-padding-left(2.5);
+	@include u-padding-right(0);
+
+	border-left: 3px solid color('accent-warm-dark');
+	margin-block-start: units(3);
+	margin-block-end: units(3);
+
+	@include at-media('tablet') {
+		margin-block-start: units(4);
+		margin-block-end: units(4);
+
+		&__align-right {
+			margin-block-start: units(3);
+			@include u-margin-left(4);
+			@include u-padding-right(4);
+			@include u-width('mobile');
+			float:right;
+		}
+
+		&__align-left {
+			margin-block-start: units(3);
+			@include u-margin-left(0);
+			@include u-margin-right(4);
+			@include u-width('mobile');
+			float:left;
+		}
+
+		&__align-center {
+			@include u-display('block');
+			@include u-margin-left(0);
+			@include u-margin-right(0);
+		}
```

Comment:
These seem to be more like modifiers than elements. Modifiers use the `--<modifier-name>` formatting. Additionally, I don't actually think we should use a modifier in this case at all, but use the same classes as we use for embeds.